### PR TITLE
Added twitter share button in footer (#23) fixes #20

### LINKED ISSUE #20: Add Share on Twitter button
Twitter Share button can be made by using anchor tag without any usage of JavaScript.

### PR DESCRIPTION
**Pull request by full-stack-king**
Added twitter share button referenced in #20

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -44,7 +44,8 @@ <h2>GAME OVER!</h2>
       </section>
     </main>
     <footer class="game-footer">
-      <p>Made for Hacktoberfest by these <strong><a href="https://github.com/itaditya/got-death-quiz#contributors" target="_blank">awesome people.</a></strong></p>
+      <p>Made for Hacktoberfest by these <strong><a href="https://github.com/itaditya/got-death-quiz#contributors" target="_blank">awesome people.</a></strong><a href="https://twitter.com/intent/tweet?text=Game%20of%20Thrones%20Death%20Quiz&url=https%3A%2F%2Fgot-death-quiz.netlify.com&hashtags=roadtohacktober,hacktoberfest" target="blank" rel="noopener noreferrer"><img
```
Comment (itaditya) [inline index.html]: Put the Twitter button below the "Reload to play again" section. Also in the text of the tweet we should have the user's score which you can get from the `store.score`.

So basically by default your twitter will be hidden just like other game-over stuff and doesn't have to contain any link. Once game over, generate the link with game score included and rest of the text you have and insert that link in the Tweet anchor tag and then un-hide it.

File: index.html [hunk 2]
```diff
@@ -44,7 +44,8 @@ <h2>GAME OVER!</h2>
...
+          style="vertical-align: bottom;margin-left: 10px;" src="/assets/tweet.png" alt="Share Page on Twitter" /></a></p>
```
Comment (itaditya) [inline index.html]: Move the styling to CSS file style.css. Also make the border-radius: 4px

File: index.html (hunk 1 above)
Comment (full-stack-king) [reply index.html]: Completed in new commit

File: index.html (hunk 2 above)
Comment (full-stack-king) [reply index.html]: Ah sure! updated the code now. 
Thank you!

File: style.css [hunk 3]
```diff
@@ -51,6 +51,16 @@ body {
   padding-left: 15px;
 }
 
+.tweet-block {
+  margin-top: 30px;
+}
+
+.game-over .tweet-icon {
```
Comment (itaditya) [inline style.css]:
```suggestion
.tweet-icon {
```

currently it does nothing as the whole tweet UI is outside game-over div.

File: style.css [hunk 4]
```diff
@@ -51,6 +51,16 @@ body {
...
+  margin-left: 10px;
+  border-radius: 4px;
+  vertical-align: bottom;
```
Comment (itaditya) [inline style.css]: remove this line.

File: style.css [hunk 5]
```diff
@@ -51,6 +51,16 @@ body {
   padding-left: 15px;
 }
 
+.tweet-block {
+  margin-top: 30px;
+}
+
+.game-over .tweet-icon {
+  margin-left: 10px;
```
Comment (itaditya) [inline style.css]: remove this line

File: index.html [hunk 6]
```diff
@@ -35,6 +35,9 @@ <h2 class="quiz-question js-quiz-question">
           <h2>GAME OVER!</h2>
           <em class="sub-text">Reload to play again...</em>
         </div>
+        <div class="tweet-block is-hidden">
```
Comment (itaditya) [inline index.html]:
```suggestion
        <div class="tweet-block js-tweet-block is-hidden">
```

File: index.html [hunk 7]
```diff
@@ -35,6 +35,9 @@ <h2 class="quiz-question js-quiz-question">
...
+          <a class="tweet" target="blank" rel="noopener noreferrer"><img class="tweet-icon" src="/assets/tweet.png" alt="Share Page on Twitter" /></a>
```
Comment (itaditya) [inline index.html]:
```suggestion
          <a class="tweet-button js-tweet" target="blank" rel="noopener noreferrer"><img class="tweet-icon" src="/assets/tweet.png" alt="Share Page on Twitter" /></a>
```

File: script.js [hunk 8]
```diff
@@ -130,6 +130,8 @@ function renderGameOver() {
   document.querySelector('.js-options-list').classList.add('is-hidden');
   document.querySelector('.js-quiz-question').classList.add('is-hidden');
   document.querySelector('.js-game-over').classList.remove('is-hidden');
+  document.querySelector('.tweet-block').classList.remove('is-hidden');
```
Comment (itaditya) [inline script.js]:
```suggestion
  document.querySelector('.js-tweet-block').classList.remove('is-hidden');
```

File: script.js [hunk 9]
```diff
@@ -130,6 +130,8 @@ function renderGameOver() {
...
+  document.querySelector('.tweet-block .tweet').href = 'https://twitter.com/intent/tweet?text=Play%20GOT%20Death%20Quiz.%20I%20have%20scored%20%23' + store.score + '&url=https%3A%2F%2Fgot-death-quiz.netlify.com&hashtags=roadtohacktober,hacktoberfest';
```
Comment (itaditya) [inline script.js]:
```suggestion
  document.querySelector('.js-tweet').href = 'https://twitter.com/intent/tweet?text=Play%20GOT%20Death%20Quiz.%20I%20have%20scored%20%23' + store.score + '&url=https%3A%2F%2Fgot-death-quiz.netlify.com&hashtags=roadtohacktober,hacktoberfest';
```

**Comment by itaditya** [comment]
@rajakannan it's working correctly, But there are some code changes, please do according to the comments and resolve the conflicts. Let me know if you need any help.

File: style.css (hunk 3 above)
Comment (full-stack-king) [reply style.css]: okay removed

File: style.css (hunk 4 above)
Comment (full-stack-king) [reply style.css]: updated

File: index.html (hunk 6 above)
Comment (full-stack-king) [reply index.html]: done

**Comment by full-stack-king** [comment]
@itaditya updated code to reflect all the suggestions. Thank you!

**Review by itaditya — APPROVED** [review]

**Comment by itaditya** [comment]
Congrats @rajakannan!

**Pull request merged**